src/transformers/modeling_tf_flaubert.py

```diff
@@ -257,9 +267,12 @@ def call(
 
             # self attention
             if not self.pre_norm:
-                attn_outputs = self.attentions[i]([tensor, attn_mask, None, cache, head_mask[i]], training=training)
+                attn_outputs = self.attentions[i](
+                    [tensor, attn_mask, None, cache, head_mask[i], output_attentions], training=training
+                )
                 attn = attn_outputs[0]
-                attentions = attentions + (attn_outputs[1],)
+                if output_attentions:
```

Comment:
Replace by:
```
if cast_bool_to_primitive(output_attentions, self.output_attentions) is True:
```
Same at lines 285 and 321.

Replace as well:
```
if output_hidden_states:
```
By
```
if cast_bool_to_primitive(output_hidden_states, self.output_hidden_states) is True:
```
At lines 265, 308, and 319